File: Solution/Frontend/reactapp/src/components/assessment/membership/entities/CandidatePerformance.ts
interface CandidatePerformance {
  id: number;
  candidateName: string;
  email: string;
  skillLevel: string;
  assessmentsTaken: number;
  averageScore: number;
  lastAssessmentDate: string;
  status: "excellent" | "good" | "average" | "needsImprovement";
}
export default CandidatePerformance;

File: Solution/Backend/java/webapi/src/main/java/com/transflower/tflcomentor/fileio/Entity/CandidatePerformance.java
package com.transflower.tflcomentor.fileio.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class CandidatePerformance {

    private int id;
    private String candidateName;
    private String email;
    private String skillLevel;
    private int assessmentsTaken;
    private int averageScore;
    private String lastAssessmentDate;
    private String status;
}